### ISSUE TITLE
Fix dynamic loss bug

### ISSUE THREAD

**Issue by mauicv** [mauicv/world-model-rl#33]
The `cross_entropy` loss function defined in `utils.py` sums over the batch dimension instead of taking the mean. Makes it hard to compare experiments with different batch sizes.
